src/ui/src/core_components/content/CoreAnnotatedText.vue

```diff
@@ -0,0 +1,248 @@
+<template>
+	<div class="AnnotatedText">
+		<span v-for="(content, i) in fields.text.value" :key="content + i">
```

Comment:
`content` can be a number and then you can have duplicated key. I suggest to cast it to be safe
```suggestion
		<span v-for="(content, i) in fields.text.value" :key="String(content) + i">
```